ontologies/Items.ttl

```diff
@@ -0,0 +1,171 @@
+	
+#######  Object properties ##### 
+  
+wi:hasIdentifier rdf:type owl:ObjectProperty ;
+	rdfs:label "hasIdentifier"@en ;
+	rdfs:comment "Relates the item to a unique system-generated identifier that governs interaction between systems and is regarded as canonical within the Wellcome data ecosystem."@en ; 
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Identifier ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasSourceIdentifier rdf:type owl:ObjectProperty ;
+	rdfs:label "hasSourceIdentifier"@en ;
+	rdfs:comment "Relates an item to an identifier used within one of the Wellcome Library's databases or other systems to refer to works or items within the Library's holdings, whether for internal technical purposes or for human citation."@
+	rdfs:domain wi:Item
+	rdfs:range wi:SourceIdentifier
+	rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasSourceIdentifierAuthority rdf:type owl:ObjectProperty ;
+	rdfs:label "hasSourceIdentifierAuthority" @en ;
+	rdfs:comment "Relates a SourceIdentifier to a particular authoritative source of such identifiers: for example, if the SourceIdentifier is MS.49 this property might indicate that this identifier has its origins in the Wellcome Library's CALM archive management system."@en ;
+	rdfs:domain wi:SourceIdentifier
+	rdfs:range wi:SourceIdentifierAuthority
+	rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasClassmark rdf:type owl:ObjectProperty ;
+	rdfs:label "hasClassmark"@en ;
+	rdfs:comment "This relates an item to a classmark, the latter being a code that relates a work to a classification of Library holdings, often but not always relating to a subject-based scheme of library classification and arrangement such as Barnard or the National Library of Medicine classification."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Classmark ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:hasLocation rdf:type owl:ObjectProperty ;
+	rdfs:label "hasLocation"@en ;
+	rdfs:comment "This relates an item to a broad division of the Library or its storage areas within which an item is held: for instance, the History of Medicine collection, or the Closed Store Archives and Manuscripts material."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Classmark ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+	
+wi:hasFileFormat rdf:type owl:Class ;
+	rdfs:label "hasFileFormat"@en ;
+	rdfs:comment "This relates an item to its file format when the item comprises a digital object."@en ;
+ 	rdfs:domain wi:Item ;
+	rdfs:range wi:FileFormat ;
+  rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+  
+#######  Data properties #####  
+	
+wi:barcode rdf:type owl:DatatypeProperty ;
+	rdfs:label "barcode"@en ;
+	rdfs:comment "Relates the item to a number string encoded on physical items by a barcode and used for the purposes of movement control."@en ;
```

Comment:
True, I'll tweak the descriptive text to allow letters; and yes, we need to be able to distinguish different sets of barcodes, I'll revisit that.